src/api/app/views/webui2/shared/user_or_groups_roles/_list.html.haml

```diff
@@ -5,8 +5,8 @@
 - records.each do |record|
   %tr{ id: "#{type}-#{valid_xml_id(record.to_s)}" }
     %td.align-middle
-      = user_with_realname_and_icon(record) if type == 'user'
-      = link_to(record.to_s, controller: 'groups', action: 'show', title: record.to_s) if type == 'group'
+      = image_tag_for(record, size: 20)
+      = link_to(display_name(record), user_or_group_show_path(record))
```

Comment:
Do all groups have icons in production? :thinking:  otherwise it is going to fail..